components/brave_wallet_ui/components/desktop/sweepstakes-banner/style.ts

```diff
@@ -0,0 +1,90 @@
+import styled from 'styled-components'
+export { CloseButton } from '../popup-modals/style'
+
+export const Card = styled.div`
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    background: ${(p) => p.theme.color.background01};
+    border-radius: 8px;
+    /* Light Theme/Dividers and outlines/divider01 */
```

Comment:
Can we remove un-need style comment's from `Figma` in this file.

Reply:
removed